release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -534,6 +534,16 @@ See xref:../operators/operator_sdk/osdk-about.adoc#osdk-about[Developing Operato
 
 In the current version, when the {product-title} performs a build and the log level is five or higher, the cluster writes the buildah version information to the build log. This information helps Red Hat Engineering reproduce bug reports. Previously, this version information was not available in the build logs.
 
+
+[discrete]
+[id="ocp-4-7-samples-operator-assistance-mirroring"]
+==== Cluster Samples Operator assistance for mirroring
+
+OpenShift Container Platform now creates a config map named `imagestreamtag-to-image` in the openshift-cluster-samples-operator namespace that contains an entry, the populating image, for each image stream tag. You can use this config map as a reference for which images need to be mirrored for your image streams to import.
```

Comment:
s/OpenShift Container Platform/{product-title}

    `openshift-cluster-samples-operator` namespace

Reply:
:facepalm: thank you